### DISCUSSION TITLE
Error interpretasi model naive bayes dengan lime

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#851] [Q&A] [answered]
<img width="778" alt="Screenshot 2024-03-24 171504" src="https://github.com/teamalgoritma/community/assets/150925524/7f120915-edb6-4d34-91a8-b1f41ecfcf1f">

Bagaimana cara mengatasi eeror diatas. 
error di proses  observasi interpretasi dari data ke 1 sampai ke 5 dari observasi data test. 

Terimakasih

**Comment by kinantytasyao** [comment]
Dear @rastitiratnasari 

Apakah data yang digunakan pada fungsi explain() dan berisi data_test sudah dicek apakah kolomnya sudah sesuai dengan data_train yang dimasukkan ke dalam fungsi lime() yang dinamai explainer, selain itu n_features seharusnya diisi dengan banyaknya jumlah variabel prediktor yang digunakan pada model yang digunakan.

**Comment by rastitiratnasari** [comment, marked as answer]
Dear Mba Kinan,

Sudah bisa mba, setelah ditelaah ada perbedaan nama variabel yg di input dengan yg diinput di LIME. Terimakasih atas feedback nya.
